src/shogun/evaluation/MachineEvaluation.h

```diff
@@ -45,29 +46,54 @@ class CMachineEvaluation: public CSGObject
 	 * @param features features to use for cross-validation
 	 * @param labels labels that correspond to the features
 	 * @param splitting_strategy splitting strategy to use
-	 * @param evaluation_criterion evaluation criterion to use
+	 * @param list_evaluation_criterion list of evaluation criterion to use
 	 * @param autolock whether machine should be auto-locked before evaluation
 	 */
 	CMachineEvaluation(CMachine* machine, CFeatures* features, CLabels* labels,
 			CSplittingStrategy* splitting_strategy,
+			CDynamicObjectArray* list_evaluation_criterion,
+            bool autolock = true);
+
+ 	/** constructor
+	 * @param machine learning machine to use
+	 * @param features features to use for cross-validation
+	 * @param labels labels that correspond to the features
+	 * @param splitting_strategy splitting strategy to use
+	 * @param evaluation_criterion evaluation criterion to use
+	 * @param autolock whether machine should be auto-locked before evaluation
+	 */
+    CMachineEvaluation(CMachine* machine, CFeatures* features, CLabels* labels,
+			CSplittingStrategy* splitting_strategy,
 			CEvaluation* evaluation_criterion, bool autolock = true);
 
 	/** constructor, for use with custom kernels (no features)
 	 * @param machine learning machine to use
 	 * @param labels labels that correspond to the features
 	 * @param splitting_strategy splitting strategy to use
-	 * @param evaluation_criterion evaluation criterion to use
+	 * @param list_evaluation_criterion list of evaluation criterion to use
 	 * @param autolock autolock
 	 */
 	CMachineEvaluation(CMachine* machine, CLabels* labels,
 			CSplittingStrategy* splitting_strategy,
-			CEvaluation* evaluation_criterion, bool autolock = true);
+			CDynamicObjectArray* list_evaluation_criterion,
+            bool autolock = true);
 
+ 	/** constructor, for use with custom kernels (no features)
+	 * @param machine learning machine to use
+	 * @param labels labels that correspond to the features
+	 * @param splitting_strategy splitting strategy to use
+	 * @param evaluation_criterion evaluation criterion to use
+	 * @param autolock autolock
+	 */
+    CMachineEvaluation(CMachine* machine, CLabels* labels,
```

Comment:
I was referring to the unusual alignment with the previous function. Seems like after adding, `set ts=4     " tabstop` to .vimrc vim was replacing tab with 4 spaces in the actual file, while in the editor everything seemed properly indented. Commenting that out worked.